core-bundle/tests/Contao/TemplateTest.php

```diff
@@ -226,4 +229,23 @@ public function testLoadsTheAssetsPackages(): void
         $template = new FrontendTemplate();
         $template->asset('/path/to/asset', 'package_name');
     }
+
+    public function testCanDumpTemplateVars(): void
+    {
+        $template = new FrontendTemplate();
+        $template->test = 1;
+
+        // Manually set the VarDumper handler so we can fetch the output buffer
+        VarDumper::setHandler(static function ($var): void {
```

Comment:
do you really need to use the `VarCloner` and `CliDumper` for the test?

```suggestion
        $output = '';

        VarDumper::setHandler(static function ($var) use (&$output): void {
            $output = $var;
        }
        
        $template->dumpTemplateVars();
        
        $this->assertSame(/.../, $output);
```

Reply:
I guess not :). Yours is a better solution anyway, since it does not depend on the formatting of the output 👍